nameko/containers.py

```diff
@@ -168,6 +166,21 @@ def __init__(self, service_cls, config):
         self._being_killed = False
         self._died = Event()
 
+    def setup_serialization(self):
+
+        serializers = self.config.get(SERIALIZERS_CONFIG_KEY, {})
+        for name, kwargs in serializers.items():
+            encoder = import_from_path(kwargs.pop('encoder'))
+            decoder = import_from_path(kwargs.pop('decoder'))
+            serialization.register(
+                name, encoder=encoder, decoder=decoder, **kwargs)
+
+        self.serializer = self.config.get(
+            SERIALIZER_CONFIG_KEY, DEFAULT_SERIALIZER)
+
+        self.accept = self.config.get(
+            ACCEPT_CONFIG_KEY, [self.serializer])
```

Comment:
I think we need to check whether the specified serializer and accept values have been registered as serializers.

Reply:
True, it should be checked and it should raise `ConfigurationError` with a clear message rather than `KeyErrors` later on and on various places.